src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -168,6 +168,11 @@ internal abstract class TexlFunction : IFunction
         /// </summary>
         public virtual bool CanSuggestInputColumns => false;
 
+        /// <summary>
+        /// Identifies which functions args to use to compose the scope type for subsequent lambda args.

```

Comment:
"which functions args" - does it mean that the 1st argument is a scope-defining argument? Or that the second argument (using a 0-based index) is a scope-defining argument? Or something else?

Reply:
This property defines how many argument, starting from 0, will be use to build the scope type.
`Join` function has `ScopeArgs => 2`, which means args `arg0` and `arg1` will be used to compose the scope type.